src/jekyll/resource.md

```diff
@@ -44,4 +44,4 @@ Type classes in scala are encoded using implicit parameters.   Due to the mechan
 
 These two implicits allow users of the library to make use of any resource class that defines a close or dispose method.  The downside is that by using structural types, the close and dispose method will be invoked via reflection.
 
-The scala-arm library also provides default implicits for common resources used from the Scala library or the Java SDK.  In particular, anything that extends `java.io.Closeable` will *not* use reflection when closing the resource.  This type class is given slightly higher priority so it will be preferred to the reflective version when resolving implicits.
```

Comment:
For me, that would be a breaking change to right the stronger `AutoCloseable` to be required

Reply:
`AutoCloseable` is weaker though?  All `Closeable`s are `AutoCloserable`s?